### PR TITLE
Define images as system properties and have community/product services

### PR DESCRIPTION
**Pull request by Sgitario**
++ These changes will use the community services by default and the services from RedHat registry in the `redhat-registry`.
++ Disable sql-app in Daily run for Native builds

### CODE REVIEW

**Review by pjgg — APPROVED** [review]

**Comment by Sgitario** [comment]
run tests

File: external-applications/src/test/java/io/quarkus/ts/external/applications/OpenShiftWorkshopHeroesIT.java [hunk 1]
```diff
@@ -42,7 +42,7 @@
     private static final String POSTGRESQL_DATABASE = "heroes-database";
     private static final int POSTGRESQL_PORT = 5432;
 
-    @Container(image = "registry.redhat.io/rhscl/postgresql-12-rhel7", port = POSTGRESQL_PORT, expectedLog = "listening on IPv4 address")
+    @Container(image = "${postgresql.10.image}", port = POSTGRESQL_PORT, expectedLog = "listening on IPv4 address")
```
Comment (rsvoboda) [inline external-applications/src/test/java/io/quarkus/ts/external/applications/OpenShiftWorkshopHeroesIT.java:45]: PostgreSQL 12 should be used

File: external-applications/src/test/java/io/quarkus/ts/external/applications/OpenShiftWorkshopVillainsIT.java [hunk 2]
```diff
@@ -43,7 +43,7 @@
     private static final String POSTGRESQL_DATABASE = "villains-database";
     private static final int POSTGRESQL_PORT = 5432;
 
-    @Container(image = "registry.redhat.io/rhscl/postgresql-12-rhel7", port = POSTGRESQL_PORT, expectedLog = "listening on IPv4 address")
+    @Container(image = "${postgresql.10.image}", port = POSTGRESQL_PORT, expectedLog = "listening on IPv4 address")
```
Comment (rsvoboda) [inline external-applications/src/test/java/io/quarkus/ts/external/applications/OpenShiftWorkshopVillainsIT.java:46]: PostgreSQL 12 should be used

File: .github/workflows/daily.yaml [hunk 3]
```diff
@@ -129,7 +129,10 @@ jobs:
           chmod +x ./quarkus-dev-cli
       - name: Test in Native mode
         run: |
-          mvn -fae -V -B -s .github/mvn-settings.xml -fae clean verify -Dnative \
+          # Skip module due to high time execution on Native
```
Comment (rsvoboda) [inline .github/workflows/daily.yaml:132]: Is long execution time trouble for daily run?

Comment (Sgitario) [reply .github/workflows/daily.yaml:132]: Yes... I'm planning to run a parallel execution for this module.

File: external-applications/src/test/java/io/quarkus/ts/external/applications/OpenShiftWorkshopVillainsIT.java (hunk 2 above)
Comment (Sgitario) [reply external-applications/src/test/java/io/quarkus/ts/external/applications/OpenShiftWorkshopVillainsIT.java]: Ok, then I will mark this scenario with `@EnabledIfSystemProperty(named = "ts.redhat.registry.enabled", matches = "true")`

File: external-applications/src/test/java/io/quarkus/ts/external/applications/OpenShiftWorkshopHeroesIT.java (hunk 1 above)
Comment (Sgitario) [reply external-applications/src/test/java/io/quarkus/ts/external/applications/OpenShiftWorkshopHeroesIT.java]: Ok, then I will mark this scenario with `@EnabledIfSystemProperty(named = "ts.redhat.registry.enabled", matches = "true")`

**Comment by Sgitario** [comment]
run tests

**Pull request merged**